cpp/data/installer/scripts/install_ubuntu_packages.sh

```diff
@@ -29,7 +29,7 @@ apt-add-repository --yes --update 'deb https://artifacts.elastic.co/packages/7.x
 
 # main installation
 apt-get --quiet --yes --allow-unauthenticated install \
-        ant apache2 apparmor-utils ca-certificates cifs-utils clang cron curl gcc git imagemagick incron jq libarchive-dev \
+        ant apache2 apparmor-utils ca-certificates cifs-utils clang clang-format-12 cron curl gcc git imagemagick incron jq libarchive-dev \
```

Comment:
If we add a package, we should also add a system_update that installs clang-format-12 on all existing ubuntu machines.

Reply:
Done!